README.md

```diff
@@ -1,43 +1,135 @@
+<!-- omit in toc -->
 # Cesium Native
 
+[![License](https://img.shields.io/:license-Apache_2.0-blue.svg)](https://github.com/CesiumGS/cesium-native/blob/main/LICENSE)
+[![Build Status](https://github.com/CesiumGS/cesium-native/actions/workflows/build.yml/badge.svg)](https://github.com/CesiumGS/cesium-native/actions/workflows/build.yml)
+[![Release](https://img.shields.io/github/v/tag/CesiumGS/cesium-native?label=release)](https://github.com/CesiumGS/cesium-native/tags)
+
+<table>
+  <tbody>
+    <tr>
+      <td>
+
+![Googleplex](doc/img/screenshot-googleplex.jpg)
+        <p align="center"><sub>The Googleplex in Mountain View, California, USA, visualized with Photorealistic 3D Tiles in Cesium for Unity.</sub></p>
+      </td>
+      <td>
+        ![New York](doc/img/screenshot-ny-metadata.jpg)
+        <p align="center"><sub>Cesium OSM Buildings in Cesium for Unreal with shading from metadata on building height and age.</sub></p>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        ![Copernicus Crater](doc/img/screenshot-copernicus-crater.jpg)
+        <p align="center"><sub>The Copernicus Crater, visualized with Cesium Moon Terrain in Cesium for Unreal.</sub></p>
+      </td>
+      <td>
+        ![San Francisco](doc/img/screenshot-san-francisco.jpg)
```

Comment:
Is this the Aerometrex San Francisco dataset? I think it is since I'm not seeing any Google attribution, and in that case, we should credit them.

Reply:
I believe so. I grabbed it from https://cesium.com/learn/omniverse/omniverse-quickstart/, where it doesn't have any attribution listed. But looking in the later tutorials, it does seem to be the Aerometrex San Francisco dataset.